### PR TITLE
[Chore] Basic project setup

### PR DESCRIPTION
**Pull request by fmontesino**
This first PR includes adding `buildSrc` plugin, setting up detekt and drafting the README structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+
+This app supports Android 5.0 and above.
+
+### Architecture
+
+TODO: Probably MVVM
+
+### Testing
+
+TODO: Probably only Unit Test. Maybe setup CI?
```
Comment (kleinsenberg) [inline README.md:26]: GitHub Actions?

File: build.gradle [hunk 2]
```diff
@@ -1,13 +1,18 @@
+apply plugin: "de.fayard.buildSrcVersions"
+
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.3.72"
     repositories {
         google()
         jcenter()
+        gradlePluginPortal()
+        maven { url 'https://jitpack.io' }
```
Comment (kleinsenberg) [inline build.gradle:9]: What are you using jitpack for?

**Review by kleinsenberg — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (fmontesino) [reply README.md:26]: That's the idea, yup! 😄

File: build.gradle (hunk 2 above)
Comment (fmontesino) [reply build.gradle:9]: Oh I think I added that one for trying out another dependency update solution and I forgot to remove it. Nice catch! 
Will delete in #3 if that's ok?

**Pull request merged**